### ISSUE TITLE
An uneven map appears when creating a map on a flat surface

### ISSUE THREAD

**Issue by nanbwrn** [leggedrobotics/elevation_mapping_cupy#99]
When I used it to build an elevation map for the quadruped robot, I found that even when the robot is on a flat surface, the created map is uneven. This causes the robot to move very unstably. Could you please suggest which parameters should be modified to improve this? Below are my settings for go1_core_param.yaml, go1_example_setup.yaml, and convex_plane_decomposition.yaml. 
Thank you all for your responses.

![1](https://github.com/user-attachments/assets/3279eb15-0f7e-4c04-9c91-65ea3adc570c)

![2](https://github.com/user-attachments/assets/09689fea-0f49-4351-9c80-74b00aeadf23)

[convex_plane_decomposition.txt](https://github.com/user-attachments/files/16324537/convex_plane_decomposition.txt)
[go1_core_param.txt](https://github.com/user-attachments/files/16324538/go1_core_param.txt)
[go1_example_setup.txt](https://github.com/user-attachments/files/16324539/go1_example_setup.txt)
[plugin_config.txt](https://github.com/user-attachments/files/16324540/plugin_config.txt)

**Comment by mktk1117** [comment]
Hi, it might be because of the initialization. 
If you disable the initialization, how it looks like? (you can put false to use_initializer_at_start)

**Comment by lucarossini-iit** [comment]
HI @mktk1117, are you speaking about setting use_initializer_at_art = false in the convex_plane_decomposition parameters or the elevation_mapping ones? I am observing that the convex_plane_decomposition creates planes and boundaries also in zones unexplored by the elevation_mapping as it is happing in the case of  @nanbwrn at the back of the robot. Is there a way to disable this or create planes at the same height of the  boundary points of the planes in the elevation map (I mean extend the orange plane on the left of the robot of the previous image in order to cover the zone on the back of the robot)? 

Thank you.

**Comment by lucarossini-iit** [comment]
Let me rephrase: is there a way to deactivate the inpaint in the GridMapPreprocessing, or is there a way to avoid drawing a convex region in correspondence of NaN and inpainted regions?

Thank you again!
